tools/doc-generator/parse/parser.go

```diff
@@ -375,11 +375,15 @@ func getFieldCustomType(t reflect.Type) (string, bool) {
 	}
 }
 
-func getFieldType(t reflect.Type) (string, error) {
+func getFieldType(t reflect.Type, rootBlocks []RootBlock) (string, error) {
```

Comment:
nit: in future, please try keep unrelated changes in a separate PR

But! Kudos for fixing this :1st_place_medal: